### PR TITLE
Kubernetes manifests from helm

### PR DESCRIPTION
**Pull request by junior**
Updated `/deploy/complete/kubernetes` as some people still using as reference to deploy and some manifest had deprecated apis.

### CODE REVIEW

**Review by mvandervliet — COMMENTED** [review]
What about the `complete/kubernetes/mushop.yaml` ?

**Comment by junior** [comment]
> What about the `complete/kubernetes/mushop.yaml` ?

generated and pushed.

We should remove the full manifest and just give instructions on how to generate.

**Comment by mvandervliet** [comment]
> > What about the `complete/kubernetes/mushop.yaml` ?
> 
> generated and pushed.
> 
> We should remove the full manifest and just give instructions on how to generate.

Possibly, but this is does stand as one of the ways to deploy - without helm that is

File: deploy/complete/kubernetes/mushop.yaml [hunk 1]
```diff
@@ -300,7 +291,7 @@ metadata:
     app.kubernetes.io/version: "0.1.0"
     app.kubernetes.io/managed-by: Helm
 data:
-  atp.init.sql: ""
```
Comment (mvandervliet) [inline deploy/complete/kubernetes/mushop.yaml:303]: can we leave this empty for this file?

File: deploy/complete/kubernetes/mushop.yaml [hunk 2]
```diff
@@ -314,7 +305,7 @@ metadata:
...
```
Comment (mvandervliet) [inline deploy/complete/kubernetes/mushop.yaml:317]: ☝️

File: deploy/complete/kubernetes/mushop.yaml [hunk 3]
```diff
@@ -389,7 +380,7 @@ metadata:
...
```
Comment (mvandervliet) [inline deploy/complete/kubernetes/mushop.yaml:392]: ☝️

File: deploy/complete/kubernetes/mushop.yaml [hunk 4]
```diff
@@ -403,7 +394,7 @@ metadata:
...
```
Comment (mvandervliet) [inline deploy/complete/kubernetes/mushop.yaml:406]: ☝️

File: deploy/complete/kubernetes/mushop.yaml [hunk 5]
```diff
@@ -662,6 +673,21 @@ spec:
       name: http
       targetPort: 3000
 ---
+# Source: mushop/templates/ingress-grafana.yaml
```
Comment (mvandervliet) [inline deploy/complete/kubernetes/mushop.yaml:676]: can omit grafana in this case

Comment (mvandervliet) [reply deploy/complete/kubernetes/mushop.yaml:676]: also, about grafana... we need to discuss

File: deploy/complete/kubernetes/mushop.yaml [hunk 6]
```diff
@@ -699,7 +725,7 @@ spec:
               containerPort: 3000
           env:
           - name: MOCK_MODE
-            value: "catalogue"
+            value: "all"
```
Comment (mvandervliet) [inline deploy/complete/kubernetes/mushop.yaml:728]: don't want all here.. only catalogue

File: deploy/complete/kubernetes/mushop.yaml [hunk 7]
```diff
@@ -1736,3 +1849,310 @@ spec:
                   name: mushop-bucket
                   key: parUrl
                   optional: true
+---
```
Comment (mvandervliet) [inline deploy/complete/kubernetes/mushop.yaml:1852]: :nit: tests not needed

**Review by mvandervliet — CHANGES_REQUESTED** [review]

File: deploy/complete/kubernetes/mushop.yaml (hunk 1 above)
Comment (junior) [reply deploy/complete/kubernetes/mushop.yaml:303]: Where you are seeing this?

Comment (junior) [reply deploy/complete/kubernetes/mushop.yaml:303]: You are asking to keep empty or asking why is empty?

Comment (mvandervliet) [reply deploy/complete/kubernetes/mushop.yaml]: saying it should stay empty... if possible

**Review by mvandervliet — APPROVED** [review]

**Pull request merged**